schemas/metadata/decoded/decoded.1.schema.json

```diff
@@ -3,6 +3,7 @@
   "id": "http://jsonschema.net",
   "properties": {
     "client_id": {
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
```

Comment:
r? @jklukas

Reply:
This seems fine. We shouldn't have any bad client_ids make it through the decoder.

Reply:
oooh, good point, will remove